Jenkinsfile

```diff
@@ -99,5 +108,44 @@ pipeline {
         """
       junit testResults: "logs/junit.xml", keepLongStdio: false
     }
+    success {
+      script {
+        if( env.OCS_OPERATOR_DEPLOYMENT == true || env.OCS_OPERATOR_DEPLOYMENT == "true" ) {
```

Comment:
if (env.OCS_OPERATOR_DEPLOYMENT in [true, 'True', 'true']){

but as Andrew suggested I think we can send all the informations and we can just pick what we need no?